comps/dataprep/redis/README.md

```diff
@@ -69,6 +95,10 @@ Please refer to this [readme](../../../vectorstores/langchain/redis/README.md).
 ## 2.2 Setup Environment Variables
 
 ```bash
+export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
+export TEI_ENDPOINT="http://${your_ip}:6006"
```

Comment:
we should make modifications for GenAIExamples, like https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml#L23

Reply:
you mean export `6006` port number as an environment variable?

Reply:
I mean the docker compose files of chatqna example should export/import the `TEI_ENDPOINT` at the line https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml#L14? so you need push a pr to modify the GenAIExample?